Source/Core/Inline.cs

```diff
@@ -400,7 +400,7 @@ void CheckRecursion(Implementation impl, Stack<Procedure /*!*/> /*!*/ callStack)
               else if (options.ProcedureInlining == CoreOptions.Inlining.Assume)
               {
                 // add assume
-                newCmds.Add(new AssumeCmd(callCmd.tok, Expr.False));
+                newCmds.Add(new AssumeCmd(callCmd.tok, Expr.True));
```

Comment:
Adding an `assume true` seems pointless. Better to remove the command altogether then.

Reply:
I think `assume false` on `inline == 0` also should be `assume true` or empty